### DISCUSSION TITLE
Followed TW CSS Forms Installation Instructions But Not Working. How to fix it?

### DISCUSSION THREAD

**Discussion by bluecrunch** [tailwindlabs/tailwindcss#4106] [Help]
Hello,

I am using Tailwind CSS and wanted to add TW CSS Forms. So to do this I followed the instructions. But I get no result (especially when working with Selects).
What I did

`npm install @tailwindcss/forms`
I added
`require('@tailwindcss/forms'),` to tailwind.config.js

But no result. I still have the default select appearance from my browser or an arrow-less version when removing appearance. 

I already checked Purge in the Config and this is empty.

So I looked at an underlining of VSC3 in the plugin-array and it says:

`module "/Users/itsme/Documents/GitHub/gs_tw/node_modules/@tailwindcss/forms/src/index"
Could not find a declaration file for module '@tailwindcss/forms'. '/Users/itsme/Documents/GitHub/gs_tw/node_modules/@tailwindcss/forms/src/index.js' implicitly has an 'any' type.
  Try npm i --save-dev @types/tailwindcss__forms if it exists or add a new declaration (.d.ts) file containing declare module @tailwindcss/forms;ts(7016)`

I looked at my node-modules and found the folder '@tailwindcss/forms' and the index.js in the required src-folder. I assume it is an issue with the naming of the folders. How should I approach it without breaking anything?

My versions are
```
"@tailwindcss/forms": "^0.3.2",
"tailwindcss": "^2.0.4",
```
And I use it with 
`"react": "^17.0.2",`

Thanks in advance.

**Comment by simonswiss** [comment]
Hey!

Are you able to share your repo to take a look, or try and create a minimal reproduction of this problem? Hard to tell without looking at that.

You mention you required the forms plugin from `tailwind.config.css`, do you mean `tailwind.config.js` in the plugins array?

**Reply by bluecrunch** [reply]
Hi Simon, 

Sure, just uploaded it. You should be able to see it here: [](https://github.com/bluecrunch/tw_forms). And yes, you are right, I meant `tailwind.config.js`. I'll edit it to not confuse anybody who may read it later.
